modules/apps/application-list/application-list-api/src/main/java/com/liferay/application/list/util/PanelCategoryRegistryUtil.java

```diff
@@ -3,8 +3,9 @@
  * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
  */
 
-package com.liferay.application.list;
+package com.liferay.application.list.util;
```

Comment:
Hey @MrJohn1911, Did you remember to check if this package is exposed on the bnd.bnd file? Also, you need to run the baseline task as well.

please check that and send it to the core-infra repo.

Thx